docs/FAQs/Dreamview_FAQs.md

```diff
@@ -112,144 +108,6 @@ each time the sim_control is restarted. ![](images/sim_control_2.0.png)
-#3  0x00007f79f3e1a3c3 in call_init (env=<optimized out>, argv=<optimized out>, argc=<optimized out>, l=<optimized out>) at dl-init.c:36
-#4  _dl_init (main_map=0x7f79f402e1c8, argc=2, argv=0x7ffc7bc11c78, env=0x7ffc7bc11c90) at dl-init.c:126
-#5  0x00007f79f3e0b29a in _dl_start_user () from /lib64/ld-linux-x86-64.so.2
-#6  0x0000000000000002 in ?? ()
-#7  0x00007ffc7bc134bd in ?? ()
-#8  0x00007ffc7bc134eb in ?? ()
-#9  0x0000000000000000 in ?? ()
-(gdb) q
-
-@in_dev_docker:/apollo$ addr2line -C -f -e /usr/local/lib/libpcl_sample_consensus.so.1.7.2 0x375bec
-double boost::math::detail::erf_inv_imp<double, boost::math::policies::policy<boost::math::policies::promote_float<false>, boost::math::policies::promote_double<false>, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy> >(double const&, double const&, boost::math::policies::policy<boost::math::policies::promote_float<false>, boost::math::policies::promote_double<false>, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy, boost::math::policies::default_policy> const&, mpl_::int_<64> const*)
-??:?
-```
-
-More info:
-
-```
-~/playground/apollo$ git rev-parse HEAD
-321bc25633fe2115e8ea4b2e68555c8c0d301b41
-
-~/playground/apollo$ docker image ls
-REPOSITORY          TAG                                    IMAGE ID            CREATED             SIZE
-apolloauto/apollo   dev-x86_64-20180320_1118               6a23927e28c3        7 days ago          6.92GB
-apolloauto/apollo   yolo3d_volume-x86_64-latest            562d2b2b5a71        11 days ago         70.6MB
-apolloauto/apollo   map_volume-sunnyvale_big_loop-latest   80aca30fa08a        3 weeks ago         1.3GB
-apolloauto/apollo   localization_volume-x86_64-latest      be947abaa650        2 months ago        5.74MB
-apolloauto/apollo   map_volume-sunnyvale_loop-latest       36dc0d1c2551        2 months ago        906MB
-
-build cmd:
-in_dev_docker:/apollo$ ./apollo.sh build_no_perception dbg
-```
-
-2. Compile pcl and copy the pcl library files to `/usr/local/lib`:
-
-Inside docker:
-
-```
-(to keep pcl in host, we save pcl under /apollo)
-cd /apollo
-git clone https://github.com/PointCloudLibrary/pcl.git
```

Comment:
We are based on pcl 1.10.1 now. And the `-mavx` problem should be avoided using the pcl patch when docker build.